tests/baselines/reference/reboundBaseClassSymbol.symbols

```diff
@@ -11,6 +11,6 @@ module Foo {
 
     interface B extends A { b: string; } 
 >B : Symbol(B, Decl(reboundBaseClassSymbol.ts, 2, 14))
->A : Symbol(A, Decl(reboundBaseClassSymbol.ts, 0, 0))
+>A : Symbol(A, Decl(reboundBaseClassSymbol.ts, 2, 7))

```

Comment:
This baseline change is incorrect -- `A` points to the type `A` at `0, 0` not the value `A` at `2, 7`. Should add a Fourslash test for this probably.